netlify.toml

```diff
@@ -0,0 +1,2 @@
+[functions]
+  node_bundler = "esbuild"
```

Comment:
Switch the function bundler to esbuild. See https://www.netlify.com/blog/2021/04/02/modern-faster-netlify-functions/. The default one seems to have some bugs on dynamic requires.

Reply:
Is this what was causing the build issues for the netlify functions?

Reply:
Yeah I believe. I look into the files that failed and it contains a dynamic require.